### DISCUSSION TITLE
CLI message not printed after go build

### DISCUSSION THREAD

**Discussion by Rosey723** [community/community#132739] [Programming Help]
### Select Topic Area

Question

### Body

I'm trying to install a project from github (https://github.com/21Bruce/resolved-bot). Has 3 steps.
Install the go programming language, this can be done by searching "golang installation"
clone this repository
run` go build` in root directory of project
I installed golang and have it in my Program Files. I have go version go1.22.5 windows/amd64. I have the repository downloaded as a zip and navigated to C:\Users\lilys\Downloads\resolved-bot-main> ( project directory ).
I ran go build and successful startup would show me a CLI welcome message. But, i don't have any message. I run go build but it just stays blank for some seconds and just shows me the same root directory line. No CLI message or any message at all. I checked and my folder has a go.mod file. I'm on windows 11. What should i do to get a successful startup?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Rosey723 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by DarioChiappello** [comment]
If you have docker installed, you can use this Dockerfile
```
FROM golang:1.20-alpine AS builder

ENV CGO_ENABLED=0 \
    GOOS=linux \
    GOARCH=amd64

WORKDIR /app

COPY go.mod go.sum ./
RUN go mod download

COPY . .

RUN go build -o resolved-server main.go

FROM alpine:latest

WORKDIR /app


COPY --from=builder /app/resolved-server .

EXPOSE 8080

ENTRYPOINT ["./resolved-server"]
```
